checks/utils/fetchers.py

```diff
@@ -94,7 +94,7 @@ def getLinesPaste(obslogText):
 
 def matchDiscord(url):
     return re.match(
-        r"(?i)\b((?:https?:(?:/{1,3}cdn\.discordapp\.com)/)(attachments/)([0-9]{18,}/[0-9]{18,}/(?:[0-9\-\_]{19}|message).txt))", url)
+        r"(?i)\b((?:https?:(?:/{1,3}cdn\.discordapp\.com)/)(attachments/)([0-9]{18,}/[0-9]{18,}/(?:[0-9\-\_]{19}|message).txt(?:\?.*)?))", url)
```

Comment:
You can probably change the added portion to this to be more specific:
```regex
(?:\?\S+\&)?
```

This will create an optional non-capturing group that starts with a literal `?` followed by one or more "any non-whitespace character"s ending with a literal `&`.

Reply:
Updated